config/as_deprecation_whitelist.yaml

```diff
@@ -14,3 +14,6 @@
 # https://projects.theforeman.org/issues/23300
 - message: 'Dangerous query method (method whose arguments are used as raw SQL) called
     with non-attribute argument(s):'
+
+# https://projects.theforeman.org/issues/27035
+- message: 'Support for MySQL is deprecated, migrate to PostgreSQL'
```

Comment:
Looks like this only works inside tests? mysql tests are failing to run with:
`/usr/local/rvm/gems/ruby-2.5.1@test_develop_pr_core-3/gems/activesupport-5.2.1/lib/active_support/lazy_load_hooks.rb:69:in block in execute_hook': DEPRECATION WARNING: You are using a deprecated behavior, it will be removed in version 1.25, Support for MySQL is deprecated, please migrate to PostgreSQL (called from <top (required)> at /home/jenkins/workspace/test_develop_pr_core/database/mysql/ruby/2.5/slave/fast/config/environment.rb:5) (ActiveSupport::DeprecationException)`

Reply:
The full message is needed, try:

```
# https://projects.theforeman.org/issues/27035
- message: 'You are using a deprecated behavior, it will be removed in version 1.25,
    Support for MySQL is deprecated, migrate to PostgreSQL'
```